src/components/Block/SpaceFormVoting.vue

```diff
@@ -26,18 +26,18 @@ const definition = {
   properties: {
     votingDelay: {
       type: 'number',
-      title: 'Voting delay',
-      examples: ['0']
+      format: 'duration',
```

Comment:
How will validation work for this duration format? AFAIK the JSON schema "duration" format is expecting a string but we have a number. `For example, P3D expresses a duration of 3 days.`

Source: https://json-schema.org/understanding-json-schema/reference/string.html

Reply:
`duration` format is defined for strings, but we use it on number/integer type fields. It's a bit ugly that we have this format conflict though, but I'm not sure if it's something we should change, can't find a better alternative name for it.